endpoints/api/repository.py

```diff
@@ -211,12 +211,6 @@ def post(self):
         type=truthy_bool,
         default=False,
     )
-    @query_param(
```

Comment:
Curious: Is there a reason for removing this get parameter?

Reply:
It's always included as part of the GET repository request, if the feature is flagged on.